students/John_Sekora/lesson05/simple.py

```diff
@@ -0,0 +1,41 @@
+# simple.py
+import datetime
+import logging
+import logging.handlers
+
+format = "%(asctime)s %(filename)s:%(lineno)-3d %(levelname)s %(message)s"
```

Comment:
`format` is a built-in method.  You'll want to choose a different variable name.